benchmarks/benchmark_functions/default_settings.py

```diff
@@ -49,3 +54,52 @@ def Template(params_):
 		template = Template
 
 	return device, diff_method, interface, params, template, measurement
+
+
+def _vqe_defaults(hyperparams):
+	"""Uses hyperparameters or defaults to construct the components of the VQE circuit for the
+	hydrogen molecule with the sto-3g basis set.
```

Comment:
does it make sense to set the benchmark for a slightly larger basis set, e.g. 6-31g ?

Reply:
Yes, we will have _light_ benchmarks and _heavy_ benchmarks for representing different users. The existing default parameter set for H2 is a light benchmark for testing the basic functionality. We will add computationally expensive examples as well.